### DISCUSSION TITLE
Welcome to pycuda Discussions!

### DISCUSSION THREAD

**Discussion by inducer** [inducer/pycuda#314] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by tschmitt1776** [comment]
I have been asked to help a grad student from Georgia Tech on an imaging issue.  He has some very nice Python code but speed is an issue.  We are attempting a port to  with pyCuda.   This is new ground for me!  We are making some progress! Eventually we will need something like Karel Adámek's FFT as the imaging uses some Fourier techniques.

Prof Klockner - I understand that the  Krokodil in Karlsruhe has closed.

tom schmitt
Atlanta, Georgia
USA
